source/agora/consensus/protocol/Nominator.d

```diff
@@ -499,7 +504,8 @@ extern(D):
     {
         scope (exit)
         {
-            this.armTaskTimer(TimersIdx.Nomination, this.nomination_interval);
+            this.armTaskTimer(TimersIdx.Nomination,
+                this.nomination_interval * (1 + this.nom_fire_count));
```

Comment:
Exponential backoff maybe ?

Reply:
Our "jitter" implementation is plain wrong, it will need to be fixed first. We can switch to using exponential backoff after that. I will have a seperate PR for it.